### ISSUE TITLE
Data Processing/Identified Individuals Bulk Image Exporter - not working

### ISSUE THREAD

**Issue by Shannon-Dubay** [pantheracorp/PantheraIDS_Issues#192]
See Laurens post on the [forum](https://pantheraids.org/forum.html?p=post%2Fcant-export-images-11260644%3Fpid%3D1321548451) (log file attached there). She has tried 3 different server-based databases

Upon testing, I CAN (unlike Lauren) export all individuals' images using server-based databases with local images. I tested S1_2017, S4_2017 ((Which is how I discovered issue #191 )), and S6_2019.
Notably, she is using more recent data than I can test with (I don't have any recent data's images to perfectly re-create her issue), so perhaps this is why she experiences an error and I don't? 

Or theres some permissions issue or other server setting?
Lauren's log file for this issue has the same Amazon error messages seen in Issue #188 , #187 , #185 , which all say something like:

> Error in .verify.JDBC.result(r, "Unable to retrieve JDBC result set for ",  : 
> Unable to retrieve JDBC result set for SELECT DISTINCT value from admin.spcs where attribute='Species' ([Amazon](500310) Invalid operation: permission denied for schema admin;)
> Error in .verify.JDBC.result(r, "Unable to retrieve JDBC result set for ",  : 
> Unable to retrieve JDBC result set for SELECT DISTINCT siteid from .cmr_trggr_tbl ([Amazon](500310) Invalid operation: syntax error at or near "." 
> Position: 29;)
> Error : $ operator is invalid for atomic vectors

**Comment by Shannon-Dubay** [comment]
I looked at the databases Lauren is specifically having trouble with (S11_2020, S5_2020 and S17_2020) in the Tables module. 

I see that none of them have the column called "Reference" in their Individual Table. (Lauren has yet to tag reference images for these surveys) Not sure if this could be impacting anything?
(The databases I tried with all have this column because they have reference images tagged).

All other data in the tables/data look correct when looking through Tables module.
